ocaml/xapi/xapi_xenops.ml

```diff
@@ -657,6 +657,18 @@ module MD = struct
       fn = 0;
     }
 
+  (** Return the virtual function (VF) for a VGPU operated in SR-IOV
+   * mode, or None otherwise
+   *)
+  let sriov_vf ~__context vgpu =
+    match vgpu.Db_actions.vGPU_pci with
```

Comment:
Where are you setting this field? So far, the PCI address only goes into the `other-config` field.

Reply:
I agree, this does not work as the value is never set.